pyproject.toml

```diff
@@ -10,23 +10,45 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.10.5"
-kubernetes = "^26.1.0"
-kopf = "1.32.1"
+python = "3.10.13"
+kubernetes = "28.1.0"
```

Comment:
on est encore sur 1.27. c'est tu compatible?

Reply:
La théorie c'est que oui. 

https://github.com/kubernetes-client/python?tab=readme-ov-file#compatibility-matrix-of-supported-client-versions